app/org/sagebionetworks/bridge/models/schedules/CronActivityScheduler.java

```diff
@@ -25,10 +26,10 @@
             
             while (shouldContinueScheduling(context, scheduledTime, scheduledActivities)) {
                 Date next = trigger.getFireTimeAfter(scheduledTime.toDate());
-                scheduledTime = new DateTime(next, context.getZone());
+                scheduledTime = new DateTime(next, DateTimeZone.UTC);
                 
                 if (shouldContinueScheduling(context, scheduledTime, scheduledActivities)) {
-                    addScheduledActivityForAllTimes(scheduledActivities, plan, context, scheduledTime);    
+                    addScheduledActivityAtTime(scheduledActivities, plan, context, scheduledTime.toLocalDate(), scheduledTime.toLocalTime());
```

Comment:
Do cron schedules not have times? If they don't, wouldn't it make sense to schedule it at midnight to be consistent with the other schedulers?